DayCare/test_daycare.py

```diff
@@ -0,0 +1,130 @@
+import pytest
+from .models import daycare
+from django.core.exceptions import ValidationError
+from django.contrib.auth.models import User
+
+EMAIL = "test@gmail.com"
+USERNAME = "testuser01"
+PASSWORD = "pass"
+NAME = "Puppies"
+DESCRIPTION = "This is the first daycare test"
+PRICE_PER_DAY = 10
+CAPACITY = 50
+AREA = "Merkaz"
+CITY = "Tel-Aviv"
+ADDRESS = "The best street 5"
+
+
+@pytest.fixture
+def create_daycare_user():
+    return daycare.create(email=EMAIL, username=USERNAME, password=PASSWORD,
+                          name=NAME, description=DESCRIPTION,
+                          price_per_day=PRICE_PER_DAY,
+                          capacity=CAPACITY, area=AREA, city=CITY, address=ADDRESS)
+
+
+@pytest.mark.django_db()
+class TestDaycareModel:
```

Comment:
There are tests that uses context manager (`with`)to catch exceptions, some are no (call `try` `catch`)
Please change it to be more consistent (use `with` where possible)

Reply:
Fixed